dev-lang/php/php-7.4.30.recipe

```diff
@@ -50,44 +50,41 @@ REQUIRES="
 	lib:libicuuc$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:libnghttp2$secondaryArchSuffix
-	lib:libonig$secondaryArchSuffix
 	lib:libpcre$secondaryArchSuffix
 	lib:libpng16$secondaryArchSuffix
 	lib:libsqlite3$secondaryArchSuffix
 	lib:libssl$secondaryArchSuffix
-	lib:libwebp$secondaryArchSuffix
 	lib:libxml2$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
-	lib:libzip$secondaryArchSuffix
+	lib:libwebp$secondaryArchSuffix
+	lib:libonig$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
+	devel:libgd$secondaryArchSuffix
 	devel:libbz2$secondaryArchSuffix
 	devel:libcrypto$secondaryArchSuffix
 	devel:libcurl$secondaryArchSuffix
 	devel:libfreetype$secondaryArchSuffix
-	devel:libgd$secondaryArchSuffix
 	devel:libicuuc$secondaryArchSuffix
 	devel:libjpeg$secondaryArchSuffix
 	devel:libnghttp2$secondaryArchSuffix
-	devel:libonig$secondaryArchSuffix
 	devel:libpcre$secondaryArchSuffix
 	devel:libpng16$secondaryArchSuffix
-	devel:libpq$secondaryArchSuffix
 	devel:libsqlite3$secondaryArchSuffix
 	devel:libssl$secondaryArchSuffix
-	devel:libwebp$secondaryArchSuffix
 	devel:libxml2$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-	devel:libzip$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libonig$secondaryArchSuffix
+	devel:libpq$secondaryArchSuffix
```

Comment:
The gd/onig/pq/webp entries need to be alphabetized as they were before.

Reply:
Fixed